android/settings.gradle

```diff
@@ -0,0 +1,5 @@
+includeBuild(new File("../../clevertap-android-sdk")) {
+        dependencySubstitution {
+            substitute(module("com.clevertap.android:clevertap-android-sdk")).using(project(":clevertap-core"))
```

Comment:
do we take the local android sources for core sdk?

Reply:
This whole file should be deleted. I has probably been left behind from the custom templates branch.

Reply:
Fixed